.buildkite/hooks/pre-exit

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-find . -user root -print0 | xargs -0 sudo rm -rf '{}'
+# Remove all root-owned files.
+# Since BuildKite's Agent is running as non-root. Root-owned files cannot be
+# deleted by the agent.
+sudo sh -c 'find . -user root -print0 | xargs -0 rm -rf "{}"'
```

Comment:
Anything with rm -rf sounds scary. Can we do it in a more controlled way? For example collect the list of file first and then delete each one of them logging them on each deletion?

Reply:
How about `rm -rfv`? So that we can log what we've deleted.